.github/workflows/workflow.yml

```diff
@@ -1,17 +1,20 @@
 name: test
-on: [push, pull_request]
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
 jobs:
   test:
     name: test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: setup node
-        uses: actions/setup-node@v2
-        with:
-          node-version: 16
-      - run: yarn
-      - name: build
-        run: yarn build
-      - name: test
-        run: yarn test
+      - name: unique
+        run: |
+          x=$(jq 'map(.name + .network + .branch + .specVersion)' templates.json)
```

Comment:
`branch` doesn't show in the console outputs, so it might still confuse user when duplicated records are presented.